cluster_management/src/main/java/com/pinterest/rocksplicator/task/BackupTaskFactory.java

```diff
@@ -77,6 +77,7 @@ public Task createNewTask(TaskCallbackContext context) {
     String storePathPrefix = "";
     long resourceVersion = jobCreationTime;
     int backupLimitMbs = DEFAULT_BACKUP_LIMIT_MBS;
+    boolean shareFilesWithChecksum = true;
```

Comment:
By default (if not present in job command), you want to keep this as true?

Reply:
Yes. by default, always enable checksum for backup/export. 
Compared to statemodel transitions, we want to be to backward consistent, so, by default, disable the checksum during backup.